pkgs/development/tools/just/default.nix

```diff
@@ -16,6 +16,12 @@ rustPlatform.buildRustPackage rec {
   nativeBuildInputs = [ installShellFiles ];
   buildInputs = lib.optionals stdenv.isDarwin [ libiconv ];
 
+  postPatch = ''
+    # this hard codes the compiler, which breaks the aarch64 in particular
+    # we rather want to set the compiler ourself
+    rm .cargo/config
+  '';
```

Comment:
Do you think we would want to remove this file in buildRustPackage for every package?

Reply:
No, it may contain useful information like linker flags.